packages/zendesk-adapter/src/filters/article/article.ts

```diff
@@ -266,7 +266,7 @@ const filterCreator: FilterCreator = ({ config, client, elementsSource, brandIdT
       const attachmentByName: Record<string, InstanceElement> = _.keyBy(
         attachments
           .filter(isInstanceElement)
-          .filter(attachment => attachment.value.name !== undefined),
```

Comment:
maybe 
```suggestion
          .filter(attachment => getName(attachment) !== undefined),
```
?
also, since `Name` really is confusing here - wdyt about renaming the function to something like `getElemName`?